style_editor.html

```diff
@@ -7043,6 +7062,7 @@
     TrExtend : TrExtend,
 
     BatteryLevel : BatteryLevel,
+    VolumeLevel : VolumeLevel,
```

Comment:
We also need a button for it.

Reply:
Confused which one. You said no buttons before and I edited them out, but now we need one? And above said no drop-down needed either?.

Reply:
Sorry for the confusion. I meant a "VolumeLevel" button in the "functions" section.
Basically an AddFunction() call.

Reply:
It's there ~line 1814